google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -299,7 +310,10 @@ def submit(
 
         _LOGGER.log_create_with_lro(self.__class__)
 
-        self._gca_resource = self.api_client.create_pipeline_job(
+        api_client = self.api_client
+        if hasattr(self._gca_resource, "template_uri"):
```

Comment:
Note to remove this when v1 support is available.

Reply:
Done.